src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -206,33 +195,14 @@ const HeatmapPlot = (props) => {
     );
   }
 
-  if (selectedGenes?.length === 0) {
-    return (
-      <Empty
-        description={(
-          <Text>Add some genes to this heatmap to get started.</Text>
-        )}
-      />
-    );
-  }
-
-  if (cellSetsHierarchy.length === 0) {
```

Comment:
This code is never reached, because this [conditional use effect](https://github.com/hms-dbmi-cellenics/ui/blob/aa71130be817de27e50e985c70ec7f3d86b143cc/src/components/data-exploration/heatmap/HeatmapPlot.jsx#L130-L142) prevents `heatmap` data from being set, causing [the conditional for platform error](https://github.com/hms-dbmi-cellenics/ui/blob/aa71130be817de27e50e985c70ec7f3d86b143cc/src/components/data-exploration/heatmap/HeatmapPlot.jsx#L176) to show up before this code is reached.